Gruntfile.js

```diff
@@ -252,20 +252,44 @@ module.exports = function (grunt) {
                     outfile: 'tmp/browser/test-runner-post-processor.html'
                 }
             },
-            postProcessorPLugin: {
-                src: ['test/browser/less/postProcessorPlugin/*.less'],
+            postProcessorPlugin: {
+                src: ['test/less/postProcessorPlugin/*.less'],
                 options: {
-                    helpers: 'test/browser/runner-postProcessorPlugin-options.js',
+                    helpers: ['test/plugins/postprocess/index.js','test/browser/runner-postProcessorPlugin-options.js'],
```

Comment:
yes, you are right. I fact this PR is work in progress. I first wrote the test for the in browser version. After that i start to integrate the  plugin test for both in browser and CLI. Only the postprocessor runs both CLI and in browser now. If you agree do the other plugin test now. All plugin tests should then read the plugin code from `test/plugins/*`